src/kernel/thread/Mybuild

```diff
@@ -25,6 +25,8 @@ module core {
 
 	depends embox.kernel.cpu.common
 	depends embox.kernel.sched.current.api
+
+	depends stack_protect
```

Comment:
Please create `abstract module` with `@DefaultImpl` wich be empty implementation of your API.
Embox support a lot of platforms. Some of them haven't support MMU and if we'll accept your PR we'll break all of ones.